### ISSUE TITLE
Wine 4.12 creates broken wineprefix

### ISSUE THREAD

**Issue by t14462** [Tk-Glitch/PKGBUILDS#237]
Something wrong with latest 4.12 - winecfg does not create ~/.wine properly

Winetricks:
wine cmd.exe /c echo '%ProgramFiles%' returned unexpanded string '%ProgramFiles%' ... this can be caused by a corrupt wineprefix (`wineboot -u` may help), by an old wine, or by not owning /home/seth/.wine

wineboot -u cant help me with it.

**Comment by rodrigo21** [comment]
I'm having the same problem when creating or updating a 64-bit prefix. On 32-bit prefix it's seems to be ok.
When accessing the Drives tab in winecfg it's says: Failed to connect to the mount manager; the drive configuration connot be edited.
Here a log:
```
000b:fixme:thread:create_user_shared_data_thread Creating user shared data update thread.
000b:fixme:winediag:start_process Wine TkG 4.12 is a testing version containing experimental patches.
000b:fixme:winediag:start_process Please don't report bugs about it on winehq.org and use https://github.com/Tk-Glitch/PKGBUILDS/issues instead.
wine: Unhandled page fault on write access to 0x00000060 at address 0x4024c5 (thread 000b), starting debugger...
000b:err:seh:start_debugger Couldn't start debugger ("winedbg --auto 10 92") (2)
Read the Wine Developers Guide on how to set up winedbg or another debugger
0009:fixme:actctx:parse_depend_manifests Could not find dependent assembly L"Microsoft.Windows.Common-Controls" (6.0.0.0)
000d:fixme:thread:create_user_shared_data_thread Creating user shared data update thread.
000d:fixme:winediag:start_process Wine TkG 4.12 is a testing version containing experimental patches.
000d:fixme:winediag:start_process Please don't report bugs about it on winehq.org and use https://github.com/Tk-Glitch/PKGBUILDS/issues instead.
000d:err:ole:marshal_object couldn't get IPSFactory buffer for interface {00000131-0000-0000-c000-000000000046}
000d:err:ole:marshal_object couldn't get IPSFactory buffer for interface {6d5140c1-7436-11ce-8034-00aa006009fa}
000d:err:ole:StdMarshalImpl_MarshalInterface Failed to create ifstub, hres=0x80004002
000d:err:ole:CoMarshalInterface Failed to marshal the interface {6d5140c1-7436-11ce-8034-00aa006009fa}, 80004002
000d:err:ole:get_local_server_stream Failed: 80004002
0010:fixme:actctx:parse_depend_manifests Could not find dependent assembly L"Microsoft.Windows.Common-Controls" (6.0.0.0)
0010:fixme:thread:create_user_shared_data_thread Creating user shared data update thread.
0010:err:winecfg:open_mountmgr failed to open mount manager err 2
0010:err:winecfg:open_mountmgr failed to open mount manager err 2
```

**Comment by Tk-Glitch** [comment]
Wine 4.12 release is broken. A hotfix is being worked on : https://www.winehq.org/pipermail/wine-devel/2019-July/148573.html

**Comment by Jetba** [comment]
Link to bug in WineHQ: https://bugs.winehq.org/show_bug.cgi?id=47464
According to the bug, this issue has been fixed in commit https://github.com/wine-mirror/wine/commit/e245df2a0c8d1270a4cc0129407072c917a0c325. Wine staging has also pulled this commit in their latest rebase.

I can confirm that 64bit prefix creation is now working with wine-tkg-git when using the latest master for wine-staging as expected.

**Comment by Kerrung** [comment]
Fixed in Wine 4.12.1.

**Comment by Tk-Glitch** [comment]
Closing since 4.12.1 hotfix fixed it.
